src/SamplesApp/UITests.Shared/Windows_UI_Input/PointersTests/NestedHandling.xaml.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using Uno.UI.Samples.Controls;
+
+namespace UITests.Windows_UI_Input.PointersTests
+{
+	[Sample(
+		"Pointers",
+		Description = "Press the pink square, move a bit and release while remaining on the pink square.")]
```

Comment:
Is it possible to do this in an automated test?

Reply:
Yep it was already in the pipeline ;)